docs/migration-guide.md

```diff
@@ -2,36 +2,59 @@
 
 All migration steps necessary in reading apps to upgrade to major versions of the Kotlin Readium toolkit will be documented in this file.
 
-<!-- ## Unreleased -->
+## 3.0.0-alpha.1
 
-## 2.4.0
+First of all, upgrade to version 2.4.0 and resolve any deprecation notices. This will help you avoid troubles, as the APIs that were deprecated in version 2.x have been removed in version 3.0.
 
-### Maven Central
+### Minimum requirements
 
-Readium is now distributed with [Maven Central](https://search.maven.org/search?q=g:org.readium.kotlin-toolkit). You must update your Gradle configuration.
+Readium 3.0 requires Kotlin 1.9.0 and Gradle 8.0.0. You should start by updating these dependencies in your application.
```

Comment:
Is this true if you use Readium as a regular external dependency?

Reply:
Yes for the Kotlin version, [see this Slack discussion](https://readium.slack.com/archives/C703MSTQU/p1623427325075300).

For Gradle 8.0, I guess it's only required when not using Maven Central, although I'm not 100% sure.